assets/csidriver.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     # Used to determine if a CSI driver was created by OCP or by 3rd party operator / helm / yaml files.
     csi.openshift.io/managed: "true"
 spec:
-  fsGroupPolicy: "None"
+  fsGroupPolicy: ReadWriteOnceWithFSType
```

Comment:
sorry - I forgot during standup but `None` is fine, because `fsgroup` on mount capability is queried by kubelet from CSI driver's capabiliies and does not need anything in CSIDriver.

Reply:
Fixed